backend/typescript/services/implementations/camperService.ts

```diff
@@ -308,6 +308,48 @@ class CamperService implements ICamperService {
     }
   }
 
+  async getCampersByChargeIdAndSessionId(
+    chargeId: string,
+    sessionId: string,
+  ): Promise<CamperDTO[]> {
+    try {
+      // eslint-disable-next-line prettier/prettier
+      const campers: Camper[] = await MgCamper.find({ chargeId, campSession: sessionId });
+
+      if (!campers || campers.length === 0) {
+        throw new Error(
+          `Campers with Charge Id ${chargeId} and Session Id ${sessionId} not found.`,
+        );
+      }
+
+      const camperDTO: CamperDTO[] = campers.map((camper) => {
+        return {
+          id: camper.id,
+          campSession: camper.campSession ? camper.campSession.toString() : "",
```

Comment:
Hmm how would determine whether or not the camp session exists? Do you mean if the given `sessionId` argument is just empty, or we actually try to GET the camp session by id and see whether or not that returns anything? It might be overkill, maybe that's a reason to keep the `throw new Error` above when `!campers`, since an invalid `sessionId` would return no campers and ultimately cause that error to get thrown anyway.